metadata.rb

```diff
@@ -4,7 +4,7 @@
 license 'Apache-2.0'
 description 'Installs/Configures auditd'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version '2.3.3'
+version '2.4.0'
```

Comment:
Don't change versions in a feature PR.

Reply:
I agree with @coderanger PRs should not bump the version number